docs/SetupGuide_DotnetOutOfProc.md

```diff
@@ -35,6 +36,28 @@ Please refer to the functions documentation [here](https://learn.microsoft.com/a
 - There's also no direct support for types inherited from underlying service SDKs, such as SqlCommand. Instead, bindings rely on strings, arrays, and serializable types, such as plain old class objects (POCOs).
 - For HTTP triggers, you must use HttpRequestData and HttpResponseData to access the request and response data. This is because you don't have access to the original HTTP request and response objects when running out-of-process.
 
+## Setup Function App
```

Comment:
This seems unrelated to this PR - could you just do it as a separate PR? It should be quick to get in since it's just a doc update.